clients/src/main/java/org/apache/kafka/clients/KafkaClient.java

```diff
@@ -81,6 +81,13 @@
     public List<ClientResponse> poll(long timeout, long now);
 
     /**
+     * Closes the connection to a particular node (if there is one).
+     *
+     * @param nodeId The id of the node
+     */
+    public void close(String nodeId);
```

Comment:
This method already existed in `NetworkClient`, I just added it to the interface.